src/typingsInstaller/nodeTypingsInstaller.ts

```diff
@@ -106,7 +106,7 @@ type ExecSync = (command: string, options: ExecSyncOptions) => string;
 export class NodeTypingsInstaller extends TypingsInstaller {
     private readonly nodeExecSync: ExecSync;
     private readonly npmPath: string;
-    readonly typesRegistry: Map<string, MapLike<string>>;
+    typesRegistry: Map<string, MapLike<string>> = undefined!;

```

Comment:
This variable is only referenced once; can we just make it potentially undefined and then assertDefined at its one usage, rather than adding `undefined!`?